src/analyze/analyze-security.c

```diff
@@ -1536,7 +1536,6 @@ static int assess(const struct security_info *info, Table *overview_table, Analy
                         return log_error_errno(r, "Failed to output table: %m");
         }
 
-        assert(weight_sum > 0);
```

Comment:
Because commit which added this line is redundant this is also redundant.

Reply:
Blah. Apparently I haven't checked carefully enough.